caravel/assets/javascripts/SqlLab/components/SqlEditorLeft.jsx

```diff
@@ -102,22 +102,27 @@ class SqlEditorTopToolbar extends React.Component {
     const tableName = tableOpt.value;
     const qe = this.props.queryEditor;
     const url = `/caravel/table/${qe.dbId}/${tableName}/${qe.schema}/`;
+
+    this.setState({ tableLoading: true });
```

Comment:
it get's set to false after the async request comes back https://github.com/airbnb/caravel/pull/1160/files#diff-2b41d6ca02b4b8163346107f847ee958R118